src/lt/object.cljs

```diff
@@ -40,33 +40,35 @@
   "Metadata of current behavior set during raise and raise-reduce"
   nil)
 
-(defn add [obj]
+(defn- add [obj]
   (swap! object-defs assoc (::type obj) obj))
 
-(defn add-behavior [beh]
+(defn- add-behavior [beh]
   (swap! behaviors assoc (:name beh) beh))
 
-(defn ->id [obj]
+(defn ->id
+  "Return id of given object"
+  [obj]
   (if (deref? obj)
     (::id @obj)
     (::id obj)))
 
-(defn ->behavior-name [beh]
+(defn- ->behavior-name [beh]
   (if (coll? beh)
     (first beh)
     beh))
 
-(defn ->behavior [beh]
+(defn- ->behavior [beh]
   (@behaviors (->behavior-name beh)))
 
-(defn ->triggers [behs]
+(defn- ->triggers [behs]
   (let [result (atom (transient {}))]
     (doseq [beh behs
             t (:triggers (->behavior beh))]
       (swap! result assoc! t (conj (or (get @result t) '[]) beh)))
     (persistent! @result)))
 
-(defn specificity-sort
+(defn- specificity-sort
```

Comment:
Agree its placement looked weird. Seems we should keep it private until we're sure of its purpose